sdk/cognitivelanguage/azure-ai-language-conversations/tests/async/test_orchestration_app_luis_response_async.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import pytest
+
+from azure.core.exceptions import HttpResponseError, ClientAuthenticationError
+from azure.core.credentials import AzureKeyCredential
+
+from testcase import GlobalConversationAccountPreparer
+from asynctestcase import AsyncConversationTest
+
+from azure.ai.language.conversations.aio import ConversationAnalysisClient
+from azure.ai.language.conversations.models import (
+    CustomConversationalTaskResult,
+    LUISTargetIntentResult,
+    OrchestratorPrediction,
+)
+
+class OrchestrationAppLuisResponseAsyncTests(AsyncConversationTest):
+
+    @GlobalConversationAccountPreparer()
+    async def test_orchestration_app_luis_response(self, endpoint, key, orch_project_name, orch_deployment_name):
+
+        # analyze query
+        client = ConversationAnalysisClient(endpoint, AzureKeyCredential(key))
+        async with client:
+            query = "Reserve a table for 2 at the Italian restaurant"
+            result = await client.analyze_conversation(
+                body={
+                    "kind": "CustomConversation",
+                    "analysisInput": {
+                        "conversationItem": {
+                            "participantId": "1",
+                            "id": "1",
+                            "modality": "text",
+                            "language": "en",
+                            "text": query
+                        },
+                        "isLoggingEnabled": False
+                    },
+                    "parameters": {
+                        "projectName": orch_project_name,
+                        "deploymentName": orch_deployment_name,
+                        "verbose": True
+                    }
+                }
+            )
+        
+            # assert - main object
+            top_project = "RestaurantIntent"
+            assert not result is None
+            assert isinstance(result, CustomConversationalTaskResult)
+            assert result.results.query == query
+            # assert - prediction type
+            assert isinstance(result.results.prediction, OrchestratorPrediction)
+            assert result.results.prediction.project_kind == "workflow"
+            # assert - top matching project
+            assert result.results.prediction.top_intent == top_project
+            top_intent_object = result.results.prediction.intents[top_project]
+            assert isinstance(top_intent_object, LUISTargetIntentResult)
+            assert top_intent_object.target_kind == "luis"
+            # assert intent and entities
+            top_intent = "RestaurantReservation.Reserve"
+            luis_result = top_intent_object.result["prediction"]
```

Comment:
Does this have to be navigated as a dictionary?

Reply:
yeah because Luis output does NOT have a fixed structure (due to different entity types it has)
this is why the autorest didn't generate a model class describing luis output